components/Form.tsx

```diff
@@ -1,11 +1,64 @@
+import Image from "next/image";
+
 const Form = () => {
-    return <form className={`flex flex-col justify-between items-center h-full w-full text-white mt-10`}>
-        Contact us to book a slot
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <button type="submit" className={`text-orange rounded bg-black pt-2 pb-2 pl-2 pr-2`}>Paw us</button>
-    </form>
-}
-export default Form;
\ No newline at end of file
+  return (
+    <div className="flex flex-col h-full w-full mt-10 pl-24 pr-24 text-black">
+      <div className="bg-form rounded-3xl p-5 flex-col flex shadow-xl">
+        <h2 className="text-3xl font-extrabold mb-4">Контактирајте нѐ!</h2>
+        <p className="mb-6">Поставете прашање или она што Ве интересира</p>
+        <div className="flex justify-between mb-6">
+          <input
```

Comment:
Input fields should have a different border color matching the design.
Additionally, the focus color of the fields is distracting. 
Check this example for how to handle this.
https://stackoverflow.com/questions/74574022/change-the-focus-border-color-in-tailwind-css